Python/Product/Analysis/Interpreter/InterpreterConfiguration.cs

```diff
@@ -73,15 +67,12 @@ public sealed class InterpreterConfiguration : IEquatable<InterpreterConfigurati
             Architecture = InterpreterArchitecture.TryParse(Read(properties, nameof(Architecture)));
             try {
                 Version = Version.Parse(Read(properties, nameof(Version)));
-            } catch (ArgumentException) {
-                Version = new Version();
-            } catch (FormatException) {
+            } catch (Exception ex) when (ex is ArgumentException || ex is ArgumentNullException || ex is FormatException) {
```

Comment:
Doesn't `ex is ArgumentException` imply that it is also `ArgumentNullException`?

Reply:
Yep